tests/wicked/before_test.pm

```diff
@@ -27,10 +27,6 @@ sub run {
     systemctl("stop " . opensusebasetest::firewall);
     systemctl("disable " . opensusebasetest::firewall);
     assert_script_run('[ -z "$(coredumpctl -1 --no-pager --no-legend)" ]');
-    record_info('INFO', 'Setting debug level for wicked logs');
-    assert_script_run('sed -e "s/^WICKED_DEBUG=.*/WICKED_DEBUG=\"all\"/g" -i /etc/sysconfig/network/config');
```

Comment:
I know that it was my idea , sorry for confusing you :( but in a fact with this PR we will decrease log level and will loose more verbose output for example in case wicked will just crash with whatever reason. So I think we need to keep this , probably better to remove `--debug`  flag from `wicked_command` since we setting it explicitly here. 
It is even smarter idea keeping in mind operational costs of typing `--debug` with every command

Reply:
ok, I will keep this and remove --debug from the command. Thanks.